docs/commands/rpc-protocol.md

```diff
@@ -10,15 +10,15 @@ The RPC protocol accepts JSON HTTP POST requests. The following are RPC commands
 ## Node RPCs
 
 !!! warning "Unconfirmed blocks returned"
-    Unless otherwise specified, RPC calls can return unconfirmed blocks and related details. In the most important cases where balances or similar details may include unconfirmed amounts, additional warnings have been included. To confirm a block, refer to _[block_confirm](#block_confirm)_, and to check for confirmation, see _[block_info](#block_info)_.
+    Unless otherwise specified, RPC calls can return unconfirmed blocks and related details. In the most important cases where balances or similar details may include unconfirmed amounts, additional warnings have been included. Refer to [Block confirmation procedures](/integration-guides/key-management/#block-confirmation-procedures) for details.
 
 ---
 
 ### account_balance 
 Returns how many RAW is owned and how many have not yet been received by **account**  
 
 --8<-- "unconfirmed-information.md"
-    The pending balance is calculated from potentially unconfirmed blocks. The account's balance is obtained from its frontier. As long as you follow the [guidelines](/integration-guides/key-management/#transaction-order-and-correctness), you can rely on the **balance** for the purposes of creating transactions for this account.
+    The pending balance is calculated from potentially unconfirmed blocks. The account's balance is obtained from its frontier. Prefer using [account_info](#account_info) for the purposes of creating transactions.
```

Comment:
For the end maybe "An atomic [account_info](#account_info) RPC call is recommended for the purposes of creating transactions."